cli/install/template.go

```diff
@@ -384,18 +384,23 @@ data:
       - source_labels: [__meta_kubernetes_namespace]
         action: replace
         target_label: namespace
+      # special case k8s' "job" label, to not interfere with prometheus' "job"
+      # label
+      # __meta_kubernetes_pod_label_conduit_io_proxy_job=foo =>
+      # k8s_job=foo
```

Comment:
I've been on both sides of the fence on this, but the fact that we have to prefix `job` to avoid conflicts makes me kinda think we should go back to prefixing all resources for consistency...

Reply:
Agree it's unfortunate that `k8s_job` ends up being a special case. I think in general it will make our code slightly more readable. An example I just ran into:
https://github.com/runconduit/conduit/issues/627#issuecomment-376656633
...where we're referring to `pod-template-hash` when talking to Kubernetes and then `k8s_pod_template_hash` immediately after when querying Prometheus.

Reply:
Also I'll note that Googling around yields a direct k8s->prom label mapping to be pretty standard, and what folks would naturally expect when querying Prometheus for Kubernetes data:
https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml#L266-L267